R/app_server.R

```diff
@@ -27,115 +27,82 @@ app_server <- function(input, output, session) {
     }
   )
 
-
   years =
     seq(
       from = input$dates[1] |> lubridate::year(),
       to = input$dates[2] |> lubridate::year()
     ) |>
     reactive()
 
+  data_source =
+    paste("Presaved CDC Data, downloaded ",  attr(presaved_CDC_data, "date")) |>
+    reactiveVal()
```

Comment:
I'm using `reactiveVal()` to create variables that exist outside of `observeEvent()` but can be changed inside of it.